cocotb/regression.py

```diff
@@ -261,6 +261,10 @@ def tear_down(self) -> None:
             self.log.info("Writing coverage data")
             self._cov.save()
             self._cov.html_report()
+        if cocotb._library_coverage is not None:
+            # move this once we have normal shutdown behavior to _sim_event
```

Comment:
nit:
```suggestion
            # TODO: move this once we have normal shutdown behavior to _sim_event
```
otherwise it looks like a description of what this line is doing, not of what needs to be done